src/client.js

```diff
@@ -1208,6 +1208,19 @@ MatrixClient.prototype.scheduleAllGroupSessionsForBackup = async function() {
     await this._crypto.scheduleAllGroupSessionsForBackup();
 };
 
+/**
+ * Marks all group sessions as needing to be backed up without scheduling
+ * them to upload in the background.
+ * @returns {Promise<int>} Resolves to the number of sessions requiring a backup.
+ */
+MatrixClient.prototype.flagAllGroupSessionsForBackup = function() {
```

Comment:
the API of this method is a bit confusing, as it seems both query and command, but I guess the reason will become clear once I see the usage for it in the next PR :)

Reply:
it is a bit confusing, but I'd argue it's no worse than `select * from x` telling you how many rows it fetched.